source/main/datatypes/node_t.h

```diff
@@ -8,7 +8,7 @@
 #pragma once
 
 #include <OgrePrerequisites.h>
-#include <OGRE/OgreVector3.h>
+#include <OgreVector3.h>
```

Comment:
But it did work without it, 20 days ago, didn't it: https://github.com/RigsOfRods/rigs-of-rods/commit/341f1fac1433c0c99c16816eea4e80a071f12ca9